client/src/components/existing-request/existing-request-display.vue

```diff
@@ -269,11 +296,34 @@ export default class ExistingRequestDisplay extends Mixins(
     return ''
   }
 
-  private get lastUpdate () {
-    if (this.nr.lastUpdate) {
-      return Moment(this.nr.lastUpdate).format('MMM Do[,] YYYY')
+  private get reviewDate () {
+    if (this.nr.waiting_time) {
+      let waitingTime = this.nr.waiting_time
+      // make sure minimum waiting time is 1
+      if (waitingTime < 1) {
+        waitingTime = 1
+      }
+      let reviewDate = new Date()
+      // add the number of days to the current date to get the review date
+      reviewDate.setDate(reviewDate.getDate() + waitingTime)
+      return Moment(reviewDate).format("MMM Do[,] YYYY") + " (" + this.nr.waiting_time + " days)"
     }
-    return ''
+    return ""
+  }
+
+  private get queueDate () {
+    if (this.nr.oldest_draft) {
+      let oldest_draft = Moment(this.nr.oldest_draft)
+      return oldest_draft.tz('America/Los_Angeles').format("MMM Do[,] YYYY")
+    }
+    return ""
+  }
+
+  private get submittedDate () {
+    if (this.nr.submittedDate) {
+      return Moment(this.nr.submittedDate).format("MMMM Do[,] YYYY, h:mm a") + " Pacific Time"
```

Comment:
This should be in Pacific tz also.
Remember, all dates from API are in UTC.
If you find yourself converting UTC to Pacific a lot, you could add a common (mixin) function somewhere to do that. Maybe [in here](https://github.com/bcgov/namerequest/blob/master/client/src/components/mixins/date-mixin.ts).

Reply:
All dates are now pacific